meilisearch-http/src/option.rs

```diff
@@ -109,16 +109,16 @@ pub struct Opt {
     pub ignore_snapshot_if_db_exists: bool,
 
     /// Defines the directory path where meilisearch will create snapshot each snapshot_time_gap.
-    #[structopt(long, env = "MEILI_SNAPSHOT_PATH")]
-    pub snapshot_path: Option<PathBuf>,
+    #[structopt(long, env = "MEILI_SNAPSHOT_DIR")]
+    pub snapshot_dir: Option<PathBuf>,
 
     /// Defines time interval, in seconds, between each snapshot creation.
     #[structopt(long, requires = "snapshot-path", env = "MEILI_SNAPSHOT_INTERVAL_SEC")]
```

Comment:
snapshot-path seems to have changed to snapshot-dir, not sure. Can youy check please? :)

Reply:
Otherwise, averything LGTM! 🎉

Reply:
I mean L116 in the `requires`

Reply:
Oh it is fixed, thanks! :)